projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/LICENSE

```diff
@@ -1 +1 @@
-See https://github.com/vmware/versatile-data-kit/blob/main/LICENSE
\ No newline at end of file
+See https://github.com/vmware/versatile-data-kit/blob/main/LICENSE
```

Comment:
Why not delete the file? Do we need it?

Reply:
Jupyterlab returns exception for not having a LICENSE  file and the extension cannot be loaded. That's why I added it again